test.js

```diff
@@ -79,12 +79,6 @@ describe('url-parse', function () {
     assume(data.href).equals('https://gist.github.com/unshiftio/url-parse');
   });
 
-  it('converts protocol to lowercase', function () {
```

Comment:
Why was this test removed? Because we still need to be sure that protocols are lowercased.